app/screens/post_options/index.test.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {mapStateToProps} from './index';
+
+import * as channelSelectors from 'mattermost-redux/selectors/entities/channels';
+import * as generalSelectors from 'mattermost-redux/selectors/entities/general';
+import * as userSelectors from 'mattermost-redux/selectors/entities/users';
+import * as commonSelectors from 'mattermost-redux/selectors/entities/common';
+import * as teamSelectors from 'mattermost-redux/selectors/entities/teams';
+import * as deviceSelectors from 'app/selectors/device';
+import * as preferencesSelectors from 'mattermost-redux/selectors/entities/preferences';
+
+channelSelectors.getChannel = jest.fn();
```

Comment:
Do you specifically need the other functions to be unmocked? Instead of importing and the manually mocking each of these, you might be able to just use `jest.mock` for each of them like `jest.mock('mattermost-redux/selectors/entities/channels');`

Reply:
Ah, right. It is breaking the other mocking. Perhaps we should be faking the state passed into the selectors instead of mocking the selectors themselves then? 0/5 since that's still not a great way of doing it because you end up with a bunch of pointless JS objects